03-seeking-help.Rmd

```diff
@@ -110,6 +110,16 @@ your issue.
 
 > #### Challenge 1 {.challenge}
 > 
+> Look at the help for the `c` function. What do you expect will
+> happen if you evaluate the following: 
```

Comment:
I think perhaps this question could be a bit more specific? Like: what type of vector do you think each of these will result in? Otherwise the type of answer you are looking for is ambiguous.